backend.py

```diff
@@ -1440,6 +1440,8 @@ def configureNetworking(mounts, admin_iface, admin_bridge, admin_config, hn_conf
 
 
     network_scripts_dir = os.path.join(mounts['root'], 'etc/sysconfig/network-scripts')
+    if not os.path.exists(network_scripts_dir):
```

Comment:
Why is this change needed? I can't see anything else here that touches network-scripts.

Having said that, it seems like both `/etc/sysconfig/network-scripts` and `/etc/sysconfig/network` are obsolete and can be removed.

In particular, the NTP server args should now go in /etc/sysconfig/chronyd.

Reply:
later in this function, it will set loopback device, 
is it still required?
https://github.com/xenserver/host-installer/blob/master/backend.py#L1452

Reply:
No, systemd sets up the loopback device early on. See https://github.com/systemd/systemd/blob/main/src/core/main.c#L2469